docs/tutorials/installation.rst

```diff
@@ -0,0 +1,38 @@
+Installation
+============
```

Comment:
Should this mention use of the container as well?

Reply:
I think so

Reply:
done